src/main/java/bio/terra/workspace/service/workspace/flight/DeleteGoogleContextFlight.java

```diff
@@ -17,18 +16,13 @@ public DeleteGoogleContextFlight(FlightMap inputParameters, Object applicationCo
     super(inputParameters, applicationContext);
 
     FlightBeanBag appContext = FlightBeanBag.getFromObject(applicationContext);
-    CrlService crl = appContext.getCrlService();
 
     RetryRule retryRule =
         new RetryRuleExponentialBackoff(
             INITIAL_INTERVALS_SECONDS, MAX_INTERVAL_SECONDS, MAX_OPERATION_TIME_SECONDS);
 
     addStep(
-        new DeleteProjectStep(crl.getCloudResourceManagerCow(), appContext.getWorkspaceDao()),
-        retryRule);
-    addStep(
-        new DeleteGoogleContextStep(
-            appContext.getWorkspaceDao(), appContext.getTransactionTemplate()),
-        retryRule);
+        new DeleteProjectStep(appContext.getCrlService(), appContext.getWorkspaceDao()), retryRule);
```

Comment:
No it can't. The retry rule is stateful and needs to be specific instance for this flight. It can be shared within the flight, since the flight is sequential and the rule is initialized each time, but it cannot be static.